File: swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/IO/Swagger/Model/SendMailRaw.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// Raw Email Object
  /// </summary>
  [DataContract]
  public class SendMailRaw {
    /// <summary>
    /// The entire email contents
    /// </summary>
    /// <value>The entire email contents</value>
    [DataMember(Name="raw_email", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "raw_email")]
    public string RawEmail { get; set; }

    /// <summary>
    /// Optional order id
    /// </summary>
    /// <value>Optional order id</value>
    [DataMember(Name="id", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "id")]
    public int? Id { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class SendMailRaw {\n");
      sb.Append("  RawEmail: ").Append(RawEmail).Append("\n");
      sb.Append("  Id: ").Append(Id).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: swagger-client/spring/src/main/java/io/swagger/model/SendMailRaw.java
package io.swagger.model;

import java.util.Objects;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonCreator;
import io.swagger.v3.oas.annotations.media.Schema;
import org.springframework.validation.annotation.Validated;
import org.openapitools.jackson.nullable.JsonNullable;
import io.swagger.configuration.NotUndefined;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonSetter;
import com.fasterxml.jackson.annotation.Nulls;
import javax.validation.Valid;
import javax.validation.constraints.*;

/**
 * Raw Email Object
 */
@Schema(description = "Raw Email Object")
@Validated
@NotUndefined
@javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.SpringCodegen", date = "2025-12-23T19:41:09.077744585-05:00[America/New_York]")


public class SendMailRaw   {
  @JsonProperty("raw_email")

  private String rawEmail = null;

  @JsonProperty("id")

  @JsonInclude(JsonInclude.Include.NON_ABSENT)  // Exclude from JSON if absent
  @JsonSetter(nulls = Nulls.FAIL)    // FAIL setting if the value is null
  private Integer id = null;


  public SendMailRaw rawEmail(String rawEmail) { 

    this.rawEmail = rawEmail;
    return this;
  }

  /**
   * The entire email contents
   * @return rawEmail
   **/
  
  @Schema(required = true, description = "The entire email contents")
  
  @NotNull
  public String getRawEmail() {  
    return rawEmail;
  }



  public void setRawEmail(String rawEmail) { 

    this.rawEmail = rawEmail;
  }

  public SendMailRaw id(Integer id) { 

    this.id = id;
    return this;
  }

  /**
   * Optional order id
   * @return id
   **/
  
  @Schema(description = "Optional order id")
  
  public Integer getId() {  
    return id;
  }



  public void setId(Integer id) { 
    this.id = id;
  }

  @Override
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    SendMailRaw sendMailRaw = (SendMailRaw) o;
    return Objects.equals(this.rawEmail, sendMailRaw.rawEmail) &&
        Objects.equals(this.id, sendMailRaw.id);
  }

  @Override
  public int hashCode() {
    return Objects.hash(rawEmail, id);
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class SendMailRaw {\n");
    
    sb.append("    rawEmail: ").append(toIndentedString(rawEmail)).append("\n");
    sb.append("    id: ").append(toIndentedString(id)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(java.lang.Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}
